lib/ruleHelper.js

```diff
@@ -111,7 +111,7 @@ RuleHelper.prototype = {
             break;
         case "MemberExpression":
             methodName = node.property.name;
-            objectName = node.object.name;
+            objectName = node.object.name || this.context.getSource(node.object);
```

Comment:
`isAllowedCallExpression` checks a string response of `codeName`. This is for `escape.methods` and `escape.taggedTemplate` config options:

I think it's fine as we are currently letting devs check for x.innerHTML = My.Sanitizer(); do we really want them to check for: x.innerHTML = Sanitize(node).do();